src/main/java/org/isf/accounting/manager/BillBrowserManager.java

```diff
@@ -143,26 +143,22 @@ public List<BillPayments> getPayments(int billID) throws OHServiceException {
 	 * @param newBill - the bill to store.
 	 * @param billItems - the list of bill's items
 	 * @param billPayments - the list of bill's payments
-	 * @return <code>true</code> if the bill has been stored, <code>false</code> otherwise.
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	@Transactional(rollbackFor=OHServiceException.class)
 	@TranslateOHServiceException
-	public boolean newBill(
+	public void newBill(
```

Comment:
according to the JPA javadoc, shouldn't this return the _saved_ Entity?

Reply:
Yes it good/should.   I was trying to make the **minimum** change as previously it **always** returned `true`.  As there are 3 entities in this method (bill, billitem, and payments) it seemed like just returning `Bill` was too simple.   I can change it if you want.

Reply:
what do you mean with "too simple"?

Reply:
It seems that 3 entities being created and returning only one seems not quite right/incomplete/etc. But without making major changes (like creating a class to return all 3) returning a Bill is probably the best.  I'll change it tomorrow.

Reply:
I've noticed now that Bill POJO has not BillItems and BillPayments in it, but it should... In this case I agree would be a major change (probably worth it) and we can tackle it in a separated issue.

Reply:
The changes could work, should we open a new issue for the 3-in-1 class ? 
(I would modify the Bill one rather than creating a new class)

Reply:
Yes; please open an issue